tipCalculator.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+    <link rel="stylesheet" href="tipCalculator.css">
+</head>
+<body>
+<div class="calculator">
+    <form class="wrap" action="">
+        <div class="title">
+        <h1>Tip Calculator</h1>
+        </div>
+        <img class="titleDrip" src="img/drips.png" alt="">
+        <div class="your-bill">
+            <h2 class="heading-top">Your bill</h2>
+            <div class="bill-wrap" style="display: flex; margin-left: 80px">
+                <label for="bill" class="label-bill">$</label>
+                <input type="number" name="bill" id="bill" class="input" >
+            </div>
+        </div>
+        <div class="split-on">
+            <h2 class="heading">Split</h2>
+            <div class="split-on-wrap">
+                <span class="btn minus">-</span>
+                <input type="number" name="people" id="people"  min="1" class="people" value="1">
```

Comment:
Instead of using a number input, it probably would have been better to use a text input and account for error handling with isNaN(). This way, you could accept change, or cents, in a bill.